mods/default/nodes.lua

```diff
@@ -1561,6 +1573,10 @@ minetest.register_node("default:sand_with_kelp", {
 		end
 
 		return itemstack
+	end,
+
+	after_destruct  = function(pos, oldnode)
+		minetest.set_node(pos, {name = "default:sand"})
```

Comment:
might be a good idea to add a [node_dig_prediction](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L4624) entry for this

Reply:
Done, thanks.